<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>925</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxPesquisa">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>15</y>
      <width>471</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>100</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="categoriaCB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>11</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Tipo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="produtoTE">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>12</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Produto</string>
     </property>
    </widget>
    <widget class="QLabel" name="textLab">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>14</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pesquisa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="procuraBtn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>90</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Procurar</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="nomeLojaLab">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>15</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nome loja</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="logoLoja">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>15</y>
      <width>71</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../../.designer/backup/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>900</width>
      <height>580</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="vistageral">
     <attribute name="title">
      <string>Vista geral</string>
     </attribute>
     <widget class="QGroupBox" name="infoProdGB">
      <property name="geometry">
       <rect>
        <x>348</x>
        <y>240</y>
        <width>540</width>
        <height>300</height>
       </rect>
      </property>
      <property name="title">
       <string>Informações do produto</string>
      </property>
      <widget class="QLabel" name="codigoLab">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>250</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Código interno</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="codigoText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLineEdit" name="nomeText">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>50</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="nomeLab">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>250</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="precoText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="precoLab">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>250</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Preço</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="quantText">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="quantLab">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>80</y>
         <width>250</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantidade</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>521</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Outros parâmetros</string>
       </property>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>500</width>
          <height>120</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="modelosGB">
      <property name="geometry">
       <rect>
        <x>348</x>
        <y>0</y>
        <width>540</width>
        <height>230</height>
       </rect>
      </property>
      <property name="title">
       <string>Modelos</string>
      </property>
      <widget class="QListView" name="modelosLV">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>520</width>
         <height>160</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QPushButton" name="adModeloBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>195</y>
         <width>40</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="edModeloBtn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>195</y>
         <width>40</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="mail-message-new"/>
       </property>
      </widget>
      <widget class="QPushButton" name="delModeloBtn">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>195</y>
         <width>40</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="linhasGB">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>0</y>
        <width>150</width>
        <height>540</height>
       </rect>
      </property>
      <property name="title">
       <string>Linhas</string>
      </property>
      <widget class="QListView" name="linhasLV">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>130</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="adMarcaBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>505</y>
         <width>40</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delMarcaBtn">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>505</y>
         <width>40</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="edMarcaBtn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>505</y>
         <width>40</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="mail-message-new"/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="produtosGB">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>0</y>
        <width>150</width>
        <height>540</height>
       </rect>
      </property>
      <property name="title">
       <string>Produtos</string>
      </property>
      <widget class="QListView" name="produtosLV">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>130</width>
         <height>471</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QPushButton" name="adItemBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>505</y>
         <width>40</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="edItemBtn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>505</y>
         <width>40</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="mail-message-new"/>
       </property>
      </widget>
      <widget class="QPushButton" name="delItemBtn">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>505</y>
         <width>40</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="pesquisa">
     <attribute name="title">
      <string>Pesquisa</string>
     </attribute>
     <widget class="QLineEdit" name="produtoTE_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>11</y>
        <width>741</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Produto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="procuraBtn_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>871</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Procurar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="categoriaCB_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Tipo</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>875</width>
        <height>450</height>
       </rect>
      </property>
      <property name="title">
       <string>Resultado da pesquisa</string>
      </property>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>850</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>420</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ordenar</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="definicoes">
     <attribute name="title">
      <string>Definições</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>875</width>
        <height>110</height>
       </rect>
      </property>
      <property name="title">
       <string>Propriedades da loja</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>35</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Nome da loja</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>35</y>
         <width>541</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>32</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Aplicar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>67</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Aplicar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>70</y>
         <width>171</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Nome do lógotipo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>70</y>
         <width>541</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>875</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>9</y>
         <width>420</width>
         <height>170</height>
        </rect>
       </property>
       <property name="title">
        <string>Informações de stock</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>32</y>
          <width>230</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Número de linhas de produtos</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>102</y>
          <width>230</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Número total de produtos</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>67</y>
          <width>230</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Número de tipos de produtos</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>191</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>65</y>
          <width>191</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>100</y>
          <width>190</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>135</y>
          <width>401</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Atualizar informações</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>445</x>
         <y>10</y>
         <width>420</width>
         <height>170</height>
        </rect>
       </property>
       <property name="title">
        <string>Operações sobre o stock</string>
       </property>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>30</y>
          <width>160</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Confirmar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>32</y>
          <width>221</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Apagar todas as linhas de produtos</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>67</y>
          <width>231</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Apagar todos os produtos das linhas</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>65</y>
          <width>160</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Confirmar</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFicheiro">
    <property name="title">
     <string>Ficheiro</string>
    </property>
   </widget>
   <addaction name="menuFicheiro"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>categoriaCB</tabstop>
  <tabstop>produtoTE</tabstop>
  <tabstop>procuraBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
